pandas/util/_decorators.py

```diff
@@ -260,7 +261,7 @@ def future_version_msg(version: str | None) -> str:
 def deprecate_nonkeyword_arguments(
     version: str | None,
     allowed_args: list[str] | None = None,
-    stacklevel: int = 2,
+    stacklevel: int = None,
```

Comment:
if you're defining it below (L308), you don't need it as argument any more, do you?

Reply:
No, I will remove it. Also should I check other files for definition of stacklevel and change them?

Reply:
you should remove it from the docstring as well